helm_deploy/values-dev.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 cronjobs:
-  activate_licence: "9,12,15,18 * * 1-5" # 9am, 12pm, 3pm, and 6pm UTC, Monday - Friday
+  activate_licences: "0 9,12,15,18 * * 1-5" # 9am, 12pm, 3pm, and 6pm UTC, Monday - Friday
```

Comment:
missed a bloomin' zero and helm lint didnt tell me and k8s just ignored it! 
Also renamed the variable